crates/bevy_ecs/src/lib.rs

```diff
@@ -1794,7 +1794,7 @@ mod tests {
     fn try_insert_batch() {
         let mut world = World::default();
         let e0 = world.spawn(A(0)).id();
-        let e1 = Entity::from_raw_u32(1).unwrap();
+        let e1 = Entity::from_raw_u32(2).unwrap();
```

Comment:
I would be more comfortable if we used a larger, more clearly arbitrary number here to avoid this problem in the future. 42 is a classic, but a large constant like 10k might be good.